### DISCUSSION TITLE
windows下能否有一个指定网卡freedom的操作

### DISCUSSION THREAD

**Discussion by JarmoHu** [v2fly/v2ray-core#756] [Ideas]
在windows下有一个网卡没有设置路由表（不想通过路由表进行网卡分流），设置freedom出口的sendthrough为该网卡IP，但是用不了。我猜测是因为还是走的默认路由。
在linux里面可以设置ip rule add from x.x.x.x table xxx，但是linux里面没有这个功能。
希望作者大大能考虑一下

**Comment by kslr** [comment]
这得有一个熟悉 windows 的人
